src/Pool.js

```diff
@@ -166,7 +170,7 @@ Pool.prototype.exec = function (method, params, options) {
   }
   else if (typeof method === 'function') {
     // send stringified function and function arguments to worker
-    return this.exec('run', [String(method), params], options);
+    return this.exec('run', [String(method), params]);
```

Comment:
What is the reason for removing `, options)` here?

Reply:
The options were added as an `exec` param to bridge the options declared on `Pool.exec` to the worker execution context. However, with the decision to move to the `signal` being injected into the worker over being passed as an argument we can simply remove these options all together since we no longer need to forward to the work to know if we should be adding the signal as function argument.

Reply:
left over from debugging will add back